Resources/Prototypes/Accents/word_replacements.yml

```
@@ -520,3 +520,84 @@
     lp-chatsan-word-10: lp-chatsan-replacement-10
     lp-chatsan-word-11: lp-chatsan-replacement-11
     # LP-ChatSanitize-End
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)